src/test/cypress/e2e/exam/test-exam/TestExamCreationDeletion.cy.ts

```diff
@@ -0,0 +1,111 @@
+import { Course } from 'app/entities/course.model';
+import { CypressExamBuilder, convertCourseAfterMultiPart } from '../../../support/requests/CourseManagementRequests';
+import dayjs from 'dayjs/esm';
+import { artemis } from '../../../support/ArtemisTesting';
+import { dayjsToString, generateUUID, trimDate } from '../../../support/utils';
+
+// Requests
+const courseManagementRequests = artemis.requests.courseManagement;
+
+// Pageobjects
+const navigationBar = artemis.pageobjects.navigationBar;
+const courseManagement = artemis.pageobjects.course.management;
+const examManagement = artemis.pageobjects.exam.management;
+const creationPage = artemis.pageobjects.exam.creation;
+const examDetailsPage = artemis.pageobjects.exam.details;
+
+const examData = {
+    title: 'exam' + generateUUID(),
+    visibleDate: dayjs(),
+    startDate: dayjs().add(1, 'day'),
+    endDate: dayjs().add(2, 'day'),
+    workingTime: 5,
+    numberOfExercises: 4,
+    maxPoints: 40,
+    startText: 'Cypress exam start text',
+    endText: 'Cypress exam end text',
+    confirmationStartText: 'Cypress exam confirmation start text',
+    confirmationEndText: 'Cypress exam confirmation end text',
+};
+
+describe('Test Exam creation/deletion', () => {
+    let course: Course;
+    let examId: number;
+
+    before(() => {
+        cy.login(artemis.users.getAdmin());
+        courseManagementRequests.createCourse().then((response) => {
+            course = convertCourseAfterMultiPart(response);
+        });
+    });
+
+    beforeEach(() => {
+        cy.login(artemis.users.getAdmin(), '/');
+    });
+
+    it('Creates a test exam', function () {
+        navigationBar.openCourseManagement();
+        courseManagement.openExamsOfCourse(course.shortName!);
+
+        examManagement.createNewExam();
+        creationPage.setTitle(examData.title);
+        creationPage.setTestMode();
+        creationPage.setVisibleDate(examData.visibleDate);
+        creationPage.setStartDate(examData.startDate);
+        creationPage.setEndDate(examData.endDate);
+        creationPage.setWorkingTime(examData.workingTime);
+        creationPage.setNumberOfExercises(examData.numberOfExercises);
+        creationPage.setExamMaxPoints(examData.maxPoints);
+
+        creationPage.setStartText(examData.startText);
+        creationPage.setEndText(examData.endText);
+        creationPage.setConfirmationStartText(examData.confirmationStartText);
+        creationPage.setConfirmationEndText(examData.confirmationEndText);
+
+        creationPage.submit().then((examResponse: Interception) => {
+            const examBody = examResponse.response!.body;
+            examId = examResponse.response!.body.id;
+            expect(examResponse.response!.statusCode).to.eq(201);
+            expect(examBody.title).to.eq(examData.title);
+            expect(examBody.testExam).to.be.true;
+            expect(trimDate(examBody.visibleDate)).to.eq(trimDate(dayjsToString(examData.visibleDate)));
```

Comment:
It looks like `dayjsToString` is not in the exact same format with the actual value according to [this test result](https://bamboo.ase.in.tum.de/browse/ARTEMIS-AETG1241-DA-7/test/case/598733369).

```
+ expected - actual
-'2023-01-13T16:35:27Z'
+'2023-01-13T16:35:27.0'
```

Probably the same is valid for `startDate` and `endDate` in the same test case.

Reply:
Thanks for the hint! I adjusted the `trimDate` function, to cut even more digits to only compare until seconds (no milliseconds). This should solve the issue.